src/NativeScript/Runtime/ReleasePool.h

```diff
@@ -13,7 +13,7 @@
 #include <typeindex>
 #include <vector>
 #include <map>
-#include <stack>
+#include <wtf/Deque.h>
```

Comment:
I couldn't move items out of `std::stack`. `WTF::Deque` has exactly that in the `takeFirst()`/`takeLast()` methods.